alerts/charts/values.yaml

```diff
@@ -3,34 +3,30 @@ global:
   ##
   commonLabels: {}
 
+  ## Additional caCert to add to the CA bundle
+  ##
+  caCert: ""
+
+  ## common greenhouse values to use
+  ##
+  greenhouse:
+    # base domain of the greenhouse organisation
+    baseDomain:
+
 # kube-prometheus-stack configuration scoped to alerts
 alerts:
 
-  ## Authentication configuration for Alertmanager
-  ##
-  auth:
-
-    ## Use existing secret for Alertmanager authentication
-    secretName: ""
-
-    ## TLS Certificate Option 1: Use Helm to automatically generate self-signed certificate.
-    autoGenerateCert:
-      enabled: true
-      # If set to true, new key/certificate is generated on helm upgrade.
-      recreate: false
-      # Cert period time in days. The default is 365 days.
-      certPeriodDays: 365
-
-    ## TLS Certificate Option 2: Use your own self-signed certificate.
-    ## The chart reads the contents of the file paths with the helm .Files.Get function.
-    ## Refer to this doc https://helm.sh/docs/chart_template_guide/accessing_files/ to understand
-    ## limitations of file paths accessible to the chart.
-    ## Path to your own PEM-encoded certificate.
-    certFile: ""
-    ## Path to your own PEM-encoded private key.
-    keyFile: ""
-    ## Path to the CA cert.
-    caFile: ""
+  # Use certmanager to generate alertmanager certs
+  certManager:
+    enabled: true
+    # self-signed root certificate
+    rootCert:
+      duration: ""  # default to be 5y
+    admissionCert:
+      duration: ""  # default to be 1y
+    issuerRef:
+    #   name: "issuer"
```

Comment:
Do we have to document this change in the [AlertManager Readme](https://github.com/cloudoperators/greenhouse-extensions/blob/73c9902387ba94800b2b232ad56880ede401fff8/alerts/README.md#L61-L90)?

Reply:
Good catch. https://github.com/cloudoperators/greenhouse-extensions/pull/637/commits/9e357e6a74896476a649cc9f320161352e3c6307